kuma/core/cache.py

```diff
@@ -1,4 +1,4 @@
 from django.core.cache import caches
 
 # just a helper to not have to redefine that all over the place
-memcache = caches['memcache']
+redis = caches['redis']
```

Comment:
I agree, I would be interested to get rid of this file too.

I was not confident to do it, I was in the mindset _"it's there for a reason"_. I think it would be a manageable change to make before merging this.